### PR TITLE
Feat: Polish general data

### LINKED ISSUE #1401: [Task]: Polish general data design
### Task description

Where we display a lot of data in rows, to better use the space, we could do this recommended by the design team

Light
image
Dark (lack design but use these colors)
image

These boxes could be applied to:
- Epoch general
- Slot general
- Account page -> block issuance tab data
- Account page -> validation tab simple data

### PR DESCRIPTION
**Pull request by brancoder**
# Description of change

closes #1401

needs https://github.com/iotaledger/explorer/pull/1370 & https://github.com/iotaledger/explorer/pull/1387 

## Type of change

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
The CI doesnt pass 🙏🏼

File: client/src/app/components/nova/address/section/account/AccountBlockIssuanceSection.tsx
Comment (brancoder) [reply client/src/app/components/nova/address/section/account/AccountBlockIssuanceSection.tsx:51]: `blockIssuerKeys` is a required field of `BlockIssuerFeature` so it should always be there.

File: client/src/app/routes/nova/SlotPage.tsx
Comment (begonaalvarezd) [reply client/src/app/routes/nova/SlotPage.tsx]: once this is ready, imo we can merge, looks amazing already 🚀

Comment (begonaalvarezd) [reply client/src/app/routes/nova/SlotPage.tsx]: it could be its own isue and PR if this escalates, but yes ideally we should unify the amount copy feature, we should review the whole usage and see where it is still needed to be improved

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**
